src/views/product-integration/component-view/components/searchable-variable-group-list/style.module.css

```diff
@@ -31,6 +45,6 @@
 	font-size: 12px;
 	text-transform: uppercase;
 	font-weight: 600;
-	margin-top: 8px;
-	margin-bottom: 24px;
+	margin-top: 24px;
+	margin-bottom: 12px;
```

Comment:
Tweaked these spacing values based on the closest final-looking design artboard I could find:

<img width="1207" alt="mobile-spacing-design" src="https://user-images.githubusercontent.com/4624598/212778865-eb750f2d-576f-454a-9dc6-0dce20a5a4b7.png">

Reply:
I know @thiskevinwang is working on mobile for this one here over here: https://github.com/hashicorp/dev-portal/pull/1533.

Reply:
💭 (no action) Looks like we're not touching the same files, but there are certainly some duplicated styles.

Reply:
Indeed - the styles for `.searchBar` have a `/* TODO */` comment that notes some duplication, for one. I can set up a quick task & work on a follow-up PR to clean those up if that'd be helpful & worth prioritizing? cc @BrandonRomano

Reply:
I think that's a good task to **scope**.  I would say tag it as low though for now, and maybe let's get to that one a little later?  I have a few more things in the queue that I would say are higher priority.

That being said, if you find yourself particularly energized to tackle that sooner rather than later, I never want to get in the way of motivation to make improvements (so you could tackle it soon if you really wanted to).